code/modules/projectiles/gun_attachables.dm

```diff
@@ -1407,6 +1407,25 @@ Defined in conflicts.dm of the #defines folder.
 	recoil_unwielded_mod = RECOIL_AMOUNT_TIER_4
 	scatter_unwielded_mod = SCATTER_AMOUNT_TIER_8
 
+/obj/item/attachable/stock/carbine/wood
+	name = "\improper L42 wooden stock"
+	desc = "A fully polished wood stock, heavy yet sturdy. Attaches to the L42A Battle Rifle. You can bash someone good with this."
+	icon_state = "l42_fullstock"
+	attach_icon = "l42fullstock_a"
+	melee_mod = 6
+	wield_delay_mod = WIELD_DELAY_SLOW
+
+/obj/item/attachable/stock/carbine/wood/Initialize()
+	..()
+	accuracy_mod = HIT_ACCURACY_MULT_TIER_8
+	recoil_mod = -RECOIL_AMOUNT_TIER_3
+	scatter_mod = -SCATTER_AMOUNT_TIER_9
+
+/obj/item/attachable/stock/carbine/wood/drill
+	name = "\improper L42 exotic wood stock"
+	desc = "A fully polished non detachable wood stock attached to select L42A ceremonial rifles. You can bash someone good with this."
```

Comment:
Missed it before but this is the same desc as parent

Reply:
diff desc this time ( if people can somehow see it)